targets/targets.json

```diff
@@ -3401,7 +3401,8 @@
             "ANALOGOUT",
             "SERIAL_ASYNCH",
             "FLASH",
-            "MPU"
+            "MPU",
+            "USBDEVICE"
```

Comment:
I am sorry, but we can't add "USBDEVICE" in the default STM32L1 configuration.
In ARM MBED, "USBDEVICE" doesn't mean that USB device feature is supported,
but means targets has a user USB connector...
You have to add this in you local custom_board.json or mbed_app.json file.

Reply:
Sorry, I was not aware the board does not have a USB port.